src/types/Select.ts

```diff
@@ -0,0 +1,5 @@
+export enum Select {
```

Comment:
As I see you have the enum, so use it when filtering your todos, and change the name as `Select` isn't meaningful, you can have different selects